packages/peregrine/lib/talons/ProductOptions/useSwatch.js

```diff
@@ -0,0 +1,13 @@
+import { useCallback } from 'react';
+
+export const useSwatch = props => {
+    const { onClick, value_index } = props;
+
+    const handleClick = useCallback(() => {
+        onClick(value_index);
```

Comment:
Previously this was being handled by a click handler applied by the `List` component which has been refactored out of `Swatch` and `Tile`.